state/azure/cosmosdb/cosmosdb.go

```diff
@@ -80,31 +67,27 @@ type CosmosOperation struct {
 
 // CosmosItem is a wrapper around a CosmosDB document.
 type CosmosItem struct {
-	documentdb.Document
 	ID           string      `json:"id"`
 	Value        interface{} `json:"value"`
 	IsBinary     bool        `json:"isBinary"`
 	PartitionKey string      `json:"partitionKey"`
 	TTL          *int        `json:"ttl,omitempty"`
-}
-
-type storedProcedureDefinition struct {
-	ID   string `json:"id"`
-	Body string `json:"body"`
+	Etag         string
 }
 
 const (
-	storedProcedureName   = "__dapr_v2__"
-	versionSpName         = "__daprver__"
 	metadataPartitionKey  = "partitionKey"
 	unknownPartitionKey   = "__UNKNOWN__"
 	metadataTTLKey        = "ttlInSeconds"
 	statusTooManyRequests = "429" // RFC 6585, 4
 	statusNotFound        = "NotFound"
 )
 
+// Value used for timeout durations
+const timeoutValue = 20
```

Comment:
Misses a duration. Is this in seconds? SHould be `20 * time.Second` then

I would also consider calling this something like `defaultTimeout`

Reply:
Also, nit, but plese put all constants in the same block (see lines 79-84 above)